src/common/RoutingList/RoutingList.js

```diff
@@ -0,0 +1,32 @@
+import {
+  Route,
+  Switch,
+} from 'react-router-dom';
+
+import {
+  ROUTING_LIST_CREATE_ROUTE,
+  ROUTING_LIST_EDIT_ROUTE,
+  ROUTING_LIST_VIEW_ROUTE,
+} from './constants';
+import { RoutingListCreate } from './RoutingListCreate';
+import { RoutingListEdit } from './RoutingListEdit';
+import { RoutingListView } from './RoutingListView';
+
+export function RoutingList() {
```

Comment:
it's not common, it's used only in order line, it's the 3rd time I ask to keep routing list in context of order lines